x-pack/plugin/ent-search/src/main/java/org/elasticsearch/xpack/application/analytics/action/DeleteAnalyticsCollectionAction.java

```diff
@@ -50,7 +58,7 @@ public ActionRequestValidationException validate() {
             ActionRequestValidationException validationException = null;
 
             if (Strings.isNullOrEmpty(collectionName)) {
-                validationException = addValidationError("collection name missing", validationException);
+                validationException = addValidationError("Collection name missing", validationException);
```

Comment:
Not sure if we want to change the case in log messages that are already in the wild? WDYT?

Reply:
Just wanted to align with other log messages. Will revert this change